synapse/config/tls.py

```diff
@@ -229,6 +230,12 @@ def default_config(self, config_dir_path, server_name, **kwargs):
             #
             # reprovision_threshold: 30
 
+            # What domain the certificate should be for. Only useful if
```

Comment:
I think we can be clearer here.

> The domain that the certificate should be for. Normally this should be the same as your matrix domain (ie, 'server_name'), but, by putting a file at 'https://<server_name>/.well-known/matrix/server', you can delegate incoming traffic to another server. If you do that, you should give the target of the delegation here.
>
> For example: if your 'server_name' is 'example.com', but 'https://example.com/.well-known/matrix/server' delegates to 'matrix.example.com', you should put 'matrix.example.com' here.

Reply:
Thanks. I wasn't happy about the documentation I had, but wasn't sure about how to improve it.